scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -83,16 +83,16 @@
           <th style="width: 10%;">Status <a href="" id="{{group_name}}_status" onclick="updateArgs('{{group_name}}','sort=status','{{sort_option}}');" class="badge" style="color:{{'black' if sort_by=='status' else 'grey'}};background-color:white;">
             <i class="fa fa-sort{{'-'+sort_order if sort_by=='status' }}" aria-hidden="true" data-bs-toggle="tooltip" title="Sort cases by 'status'"></i>
           </a></th>
-          <th style="width: 15%;">Analysis date <a href="" id="{{group_name}}_analysis_date" onclick="updateArgs('{{group_name}}','sort=analysis_date','{{sort_option}}');" class="badge" style="color:{{'black' if sort_by=='analysis_date' else 'grey'}};background-color:white;">
+          <th style="width: 12%;">Analysis date <a href="" id="{{group_name}}_analysis_date" onclick="updateArgs('{{group_name}}','sort=analysis_date','{{sort_option}}');" class="badge" style="color:{{'black' if sort_by=='analysis_date' else 'grey'}};background-color:white;">
             <i class="fa fa-sort{{'-'+sort_order if sort_by=='analysis_date' }}" aria-hidden="true" data-bs-toggle="tooltip" title="Sort by analysis date"></i>
           </a></th>
-          <th style="width: 15%;">Link</th>
-          <th style="width: 15%;">Default panel</th>
-          <th style="width: 5%;">Sequencing</th>
-          <th style="width: 10%;">Track <a href="" id="{{group_name}}_track" onclick="updateArgs('{{group_name}}','sort=track','{{sort_option}}');" class="badge text-body" style="color:{{'black' if sort_by=='track' else 'grey'}};background-color:white;">
+          <th style="width: 14%;">Link</th>
```

Comment:
I see that now the panel cell has space, but on a small screen like my laptop, the cell before (link) is the one that splits the case info onto 2 lines.

<img width="439" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/d4641345-6b78-4f03-ad17-88c31d430626">

Can't we split the space like 50-50 between these 2 cells. like 20% and 20% ?

Reply:
Gotcha, yes, that makes sense if the screen is small enough and the panel string is shorter - note no "omim-auto panelapp-green + 29" on your side. I'm slightly regretting the "+ N" as the others are not really default and already caused a minor confusion. But, on the positive side they do highlight which were ordered outside the master-list.
I'll tweak!

Reply:
So, I rebalanced again, giving link a little more and shrinking some of the headings to make them not auto-expand to more space. This is a little silly, since it is not perfect on small screens, and not large ones either. Adaptive layout here would make more sense, but with a prio to conserving most cases on one line for the layout, but allowing some to wrap to more (like a high TeX "badness") rather than trying to fit all leaving a lot of unused space. Anyway, this now kind of "works" down to below 1400px, unless the cases are all with multiple long named panels.
![Screenshot 2023-09-13 at 17 47 10](https://github.com/Clinical-Genomics/scout/assets/758570/d6872d92-5404-4f3a-b4b8-510dd6c8d848)